### PR TITLE
Add HTML meta page descriptions for the most important landing pages

### PR DESCRIPTION
**Pull request by helena**
#24717

### CODE REVIEW

File: akka-docs/src/main/paradox/extending-akka.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: how to extend Akka with Akka Extensions.
```
Comment (raboof) [inline akka-docs/src/main/paradox/extending-akka.md]:
```suggestion
project.description: How to extend Akka with Akka Extensions.
```

**Review by raboof — APPROVED** [review]
This is great!

File: akka-docs/src/main/paradox/io-tcp.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Using TCP with Akka.
+---
```
Comment (johanandren) [inline akka-docs/src/main/paradox/io-tcp.md]: I don't think we should see this is a landing page, it's nowadays more like the low level underpinnings of other stuff, if people ask about doing TCP we'll very likely direct them to the stream tcp APIs instead. 

Maybe just not put a meta on this at all, or something like "Low level API for using TCP  with classic actors"?

File: akka-docs/src/main/paradox/multi-jvm-testing.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Multi JVM testing with Akka.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/multi-jvm-testing.md]: "Multi JVM testing of distributed systems built with Akka"?

File: akka-docs/src/main/paradox/multi-node-testing.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Multi node testing with Akka.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/multi-node-testing.md]: Same as for multi-jam

File: akka-docs/src/main/paradox/remoting-artery.md [hunk 5]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Akka remoting.
+---
```
Comment (johanandren) [inline akka-docs/src/main/paradox/remoting-artery.md]: Also not quite a landing page, we'd want people to go to the cluster side and only ever use remoting if they really know what they are doing (tm). 

So perhaps "Details about the underlying remoting module for Akka Cluster."

File: akka-docs/src/main/paradox/io-udp.md [hunk 6]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Using UDP with Akka.
+---
```
Comment (johanandren) [inline akka-docs/src/main/paradox/io-udp.md]: Kind of the same as for TCP but here we don't have a stream API.

Maybe "Low level API for using UDP with classic actors"?

**Review by patriknw — APPROVED** [review]
LGTM, nice!

File: akka-docs/src/main/paradox/serialization.md [hunk 7]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Serialization with Akka.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/serialization.md]: in/for vs with?

Or perhaps "Serialization APIs built into Akka"

File: akka-docs/src/main/paradox/stream/index.md [hunk 8]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Streaming events with Akka Streams using stream graphs, stream flows failure handling and backpressure.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/stream/index.md]: Perhaps take the text from the akka.io/docs page (or something like it) instead
"An intuitive and safe way to do asynchronous, non-blocking backpressured stream processing."

File: akka-docs/src/main/paradox/typed/cluster-sharding.md [hunk 9]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Shard a clustered compute process across the network with location transparency message routing using Akka Cluster Sharding.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/cluster-sharding.md]: "with location transparent message routing"

File: akka-docs/src/main/paradox/typed/fsm.md [hunk 10]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Finite State Machine (FSM) with Akka Actors.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/fsm.md]: "Machines"?

File: akka-docs/src/main/paradox/typed/persistence-snapshot.md [hunk 11]
```diff
@@ -1,3 +1,6 @@
+---
+project.description: Append only event logs, snapshots and recovery with Akka event sourced actors.
+---
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/persistence-snapshot.md]: Not really a landing page?

**Review by johanandren — APPROVED** [review]
LGTM with some nits and opinions.

It would be good if the persistence query had one as well.

File: akka-docs/src/main/paradox/typed/persistence-snapshot.md (hunk 11 above)
Comment (helena) [reply akka-docs/src/main/paradox/typed/persistence-snapshot.md]: I agree, though I thought snapshotting a common topic one would search for. WDYT?

File: akka-docs/src/main/paradox/multi-node-testing.md (hunk 4 above)
Comment (helena) [reply akka-docs/src/main/paradox/multi-node-testing.md]: Is that a new type of berry?

File: akka-docs/src/main/paradox/io-tcp.md (hunk 2 above)
Comment (helena) [reply akka-docs/src/main/paradox/io-tcp.md]: It seems users eventually want to know more about UDP/TCP options etc. so I thought it should get something for search-ability.

**Pull request merged**